### DISCUSSION TITLE
[Feature request] Add title line limit in the list of notes

### DISCUSSION THREAD

**Discussion by mxng** [standardnotes/forum#2600] [Suggestions]
Currently there is no limit on how many lines of the title can be shown in the notes list section.

So when notes have really long titles the notes' titles take a lot of space in the notes list section.

Could you add an option so we can choose to either limit the title lines shown to two or leave it as it currently is now?

If it's more work than it's worth then it's alright!

**Comment by atmoio** [comment]
I'll keep this one in mind.
